frontend/class-opengraph.php

```diff
@@ -572,14 +561,26 @@ public function type( $echo = true ) {
 	 *
 	 * @param string|boolean $image Optional image URL.
 	 */
+
 	public function image( $image = false ) {
 		$opengraph_images = new WPSEO_OpenGraph_Image( $this->options, $image );
 
-		foreach ( $opengraph_images->get_images() as $img ) {
+		/**
+		 * Filter: 'wpseo_opengraph_image' - Allow changing the OpenGraph images of the page
+		 *
+		 */
+		$images = apply_filters( 'wpseo_opengraph_image', $opengraph_images->get_images() );
```

Comment:
This filter is already used. Please rename it into `wpseo_opengraph_images`.

Can you document the attribute you are passing, like the next example.
```
        /**
		 * Filter: 'wpseo_opengraph_images' - Allow changing the OpenGraph images of the page
		 *
		 * @api array $images The OpenGraph images.
		 */
```